tools/migration/internal/repo_fs_wrangler_test.go

```diff
@@ -1,13 +1,14 @@
 package internal_test
 
 import (
+	"io/ioutil"
 	"os"
+	"path"
 	"regexp"
 	"testing"
 
-	"github.com/golangci/golangci-lint/pkg/fsutils"
-	ast "github.com/stretchr/testify/assert"
-	req "github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
```

Comment:
this was a bad Goland auto-import that I missed in the previous PR.  Thanks to @rkowalick and @acruikshank for catching it